packages/transition-backend/file/__tests__/definitions.test.ts

```diff
@@ -0,0 +1,94 @@
+import fs from 'fs';
+
+// Language configuration - TODO: this should be moved to a separate config file
+const languages = [
+    'en',
```

Comment:
Add a comment that say that the first one is the reference language

Reply:
added